rttest/src/rttest.cpp

```diff
@@ -268,13 +288,15 @@ int Rttest::read_args(int argc, char ** argv)
   size_t sched_policy = SCHED_RR;
   // -m,--memory-size
   size_t stack_size = 1024 * 1024;
+  // -d,--prefault-dynamic-memory-size
+  size_t prefault_dynamic_size = 8589934592UL;  // 8GB
```

Comment:
This line introduced a compiler warning on armhf: https://ci.ros2.org/view/packaging/job/packaging_linux-armhf/392/warningsResult/

Reply:
Will take a look.

Reply:
See #90.